src/listener.ts

```diff
@@ -8,36 +8,182 @@ import { writeFromReadableStream } from './utils'
 const regBuffer = /^no$/i
 const regContentType = /^(application\/json\b|text\/(?!event-stream\b))/i
 
-export const getRequestListener = (fetchCallback: FetchCallback) => {
-  return async (
-    incoming: IncomingMessage | Http2ServerRequest,
-    outgoing: ServerResponse | Http2ServerResponse
-  ) => {
-    const method = incoming.method || 'GET'
-    const url = `http://${incoming.headers.host}${incoming.url}`
+const globalResponse = global.Response
+const responsePrototype: Record<string, any> = {
+  getResponseCache() {
+    this.__cache = undefined
+    return (this.responseCache ||= new globalResponse(this.body, this.init))
+  },
+  get body() {
```

Comment:
This would make a huge difference for devs migrating from Express to Hono in the next few weeks. 🚀 

Would it be possible to also support a built-in middleware such as body-parser (or a how-to guide) via config? cc @yusukebe 

Thanks!